migrations/000011_alter_workloads_table_workloadtype.down.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TYPE workloadtype RENAME TO workloadtype_old;
+
+CREATE TYPE workloadtype AS ENUM ('deployment', 'deploymentconfig', 'replicaset', 'replicationcontroller', 'statefulset', 'daemonset');
```

Comment:
Noticed the same order of values, [here](https://github.com/RedHatInsights/ros-ocp-backend/blob/main/migrations/000003_create_workloads_table.up.sql#L1). I don't think this will lead to issue resolution.

Reply:
As discussed, change is now introduced in existing migration file.